plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -250,6 +255,10 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
           "In \"light\" mode all incoming blocks headers will be fully validated; transactions in those validated blocks will be trusted \n")
          ("disable-ram-billing-notify-checks", bpo::bool_switch()->default_value(false),
           "Disable the check which subjectively fails a transaction if a contract bills more RAM to another account within the context of a notification handler (i.e. when the receiver is not the code of the action).")
+#ifdef EOSIO_DEVELOPER
+         ("disable-all-subjective-mitigations", bpo::bool_switch()->default_value(false),
+          "Disable all subjective mitigations checks in the entire codebase.")
+#endif
```

Comment:
Here?:

```
            bool                     disable_all_subjective_mitigations = false; //< for testing purposes only
```

If here, what should I write exactly? 

Draft:

```
//< for developer & testing purposes, can be configured using `disable-all-subjective-mitigations` when `EOSIO_DEVELOPER` build option is provided
```

Otherwise, I'm not sure where...

Reply:
Yes, That works.